site/components/header/index.js

```diff
@@ -1,33 +1,45 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+// @flow
```

Comment:
I'm pretty sure that this line (`// @flow`) needs to be at the very top of the file.